### DISCUSSION TITLE
可以新增工单详情对外的接口吗？

### DISCUSSION THREAD

**Discussion by smartsuri** [hhyo/Archery#2219] [Q&A]
工单详情接口: /sqlworkflow/detail_content/?workflow_id=xxx 
外部访问需要先登录才能调用，是否可以新增一个不需要登录，就可以获取工单详情(只需要获取sql工单执行的具体信息) 对外的接口？

**Comment by nick2wang** [comment]
外部请求工单信息肯定是需要鉴权的，如果想要完全放开接口，可以自行做二次开发
